### ISSUE TITLE
[bug] incorrect imports in test scripts

### ISSUE THREAD

**Issue by ChaiBapchya** [aws/deep-learning-containers#268]
Checklist
- [x] I've prepended issue tag with type of change: [bug]
- [x] (If applicable) I've attached the script to reproduce the bug
- [x] I've built my own container based off DLC (and I've attached the code used to build my own image)
```
pytest integration/local/test_mnist_training.py --region us-west-2 --docker-base-name 968277166688.dkr.ecr.us-west-2.amazonaws.com/beta-mxnet-training-bapac --tag 1.6.0-gpu-py36-cu101-ubuntu16.04-example-2020-06-11-00-32-53 --aws-id 968277166688 --instance-type p2.xlarge
============================================================================================================================= test session starts ==============================================================================================================================
platform linux -- Python 3.6.9, pytest-4.5.0, py-1.8.1, pluggy-0.11.0
rootdir: /home/ubuntu/deep-learning-containers/test/sagemaker_tests/mxnet/training
plugins: requests-mock-1.7.0, xdist-1.31.0, timeout-1.3.4, rerunfailures-8.0, forked-1.1.3, cov-2.9.0
collected 0 items / 1 errors

==================================================================================================================================== ERRORS ====================================================================================================================================
__________________________________________________________________________________________________________ ERROR collecting integration/local/test_mnist_training.py ___________________________________________________________________________________________________________
ImportError while importing test module '/home/ubuntu/deep-learning-containers/test/sagemaker_tests/mxnet/training/integration/local/test_mnist_training.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
integration/local/test_mnist_training.py:20: in <module>
    import local_mode_utils
E   ModuleNotFoundError: No module named 'local_mode_utils'
------------------------------------------------------------------------------------------------------------------------------- Captured stderr --------------------------------------------------------------------------------------------------------------------------------
WARNING:root:pandas failed to import. Analytics features will be impaired or broken.
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
=========================================================================================================================== 1 error in 0.10 seconds ============================================================================================================================
```

**Comment by Satish615** [comment]
import issues are fixed in pr #210 . This will be merged today

**Comment by Satish615** [comment]
The bugs are fixed in #316 but currently this pr has a timeout issue for test_distributed_mnist. we are trying to find the root cause for the timeout in this test. we will merge the pr once the timeout issue is solved.

**Comment by TusharKanekiDey** [comment]
The issue is solved with PR #316.
